scrapers/spiders/us/mo/bills.py

```diff
@@ -234,6 +244,10 @@ def parse_house_sponsors(self, response, bill_id, bill):
                 classification = 'co-sponsor'
             elif sponsor_type == 'Sponsor':
                 classification = 'primary'
+            elif 'House' in sponsor_type:
```

Comment:
I don't think these four lines are correct - what led you to add them?

I looked through the `sponsor_type` values here recently and I don't remember seeing `House` and `Senate` in the values. Secondly, this method is adding sponsorships, and `classification` for a sponsorship classifies the nature of the sponsoring relationship (eg `co-sponsor` vs `primary`), not the the nature of the sponsor themself. There *is* a [param that can be used to specify the chamber of the sponsor](https://github.com/openstates/scrapy-test/blob/dfa876fa6fbd3e9445044c907f665170d15c72cc/core/scrape/bill.py#L109) though.

Reply:
@jessemortenson, sorry for my bad. I've added them myself without discussing with you.
- https://documents.house.mo.gov/xml/231-HB3.xml
In this bill, the type of some sponsors are 'HouseConferee', 'SenateConferee'. But these sponsors are not in the main website. Please let me know your thought.

Reply:
So I got such validation error.
`Failed validating 'minLength' in schema['properties']['sponsorships']['items']['properties']['classification']:
    {'minLength': 1, 'type': 'string'}

On instance['sponsorships'][14]['classification']:
    ''`

Reply:
Ah, gotcha - I ported your logic back into the old scraper framework so we could get the benefits ASAP. When doing that I fixed this issue, you can grab code from here if you want: https://github.com/openstates/openstates-scrapers/blob/71297919999afd380b232b4e9475c897db1e255d/scrapers/mo/bills.py#L419